boards/arm/mimxrt1060_evk/mimxrt1060_evk.dts

```diff
@@ -92,6 +95,10 @@
 arduino_serial: &lpuart3 {};
 
 &flexspi {
+	status = "okay";
+	ahb-prefetch;
+	ahb-read-addr-opt;
+	rx-clock-source = <1>;
 	reg = <0x402a8000 0x4000>, <0x60000000 DT_SIZE_M(8)>;
```

Comment:
I don't know yet. Since we're close to closing the 2.6 merge window, I wanted to get initial support for mcuboot in now and revisit multiple device support in the 2.7 release cycle. Do you have multiple memory devices on the same bus on your custom board?

Maybe we can do something like this?
```
reg =	<0x402a8000 0x4000>,		/* FlexSPI peripheral registers */
	<0x60000000 DT_SIZE_M(8)>,	/* device 0 */
	<0x60800000 DT_SIZE_M(8)>,	/* device 1 */
	<0x61000000 DT_SIZE_M(8)>,	/* device 2 */
	<0x61800000 DT_SIZE_M(8)>;	/* device 3 */
```

Reply:
I don't have it on any of my devices, it was a general question. 

Your proposal seems reasonable, however it still feels like there is a big disconnect between the controller and devices, especially if they are scattered across dts/dtsi files.